src/asciidoc/docs/guidelines/persistSessionData.adoc

```diff
@@ -0,0 +1,60 @@
+= Persisting data across sessions
+:page-slug: persist-session-data
+:page-description: How to persist data that doesn't expire with the session.
+
+Data can be stored temporarily using window.sessionStorage.
```

Comment:
This paragraph still has a lot of passive voice that we can remove. Consider something like (might need some word-smithing and an accuracy check):

Some templates need to store data about the user's session, such as a user's name or game score.
To store data temporarily, you can use `window.sessionStorage`.
However, data stored in this way expires when the session ends.
If you need to persist data across sessions, you can use `window.localStorage`.

Reply:
Done